lib/api.js

```diff
@@ -146,6 +163,52 @@ PlexAPI.prototype._request = function _request(relativeUrl) {
     return deferred.promise;
 };
 
+PlexAPI.prototype._post = function perform(relativeUrl) {
```

Comment:
`_post()` looks very similar to `_request()`. Maybe `_request()` could be refactored to handle both GET and POST?